app/src/test/java/org/pesmypetcare/mypetcare/controllers/washes/TestTrObtainAllPetWashes.java

```diff
@@ -0,0 +1,38 @@
+package org.pesmypetcare.mypetcare.controllers.washes;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.pesmypetcare.mypetcare.controllers.washes.TrObtainAllPetWashes;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [No need to import a type that lives in the same package](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care/pullRequest?prid=5561369)